crates/tx5-go-pion/src/peer_con.rs

```diff
@@ -95,6 +95,7 @@ impl From<&AnswerConfig> for GoBufRef<'static> {
 
 pub(crate) struct PeerConCore {
     peer_con_id: usize,
+    con_state: PeerConnectionState,
```

Comment:
Somewhat torn about this. We shouldn't try to close connections that aren't in `connected` or `failed` but I'm not sure we can access this object while the connection is being established. We also have the option to ask for the connection state by calling Go, but that requires an unsafe call to lookup when we've already been told what the state is. Either way, I'm not enforcing this check for now, just warning if we are closing connections we shouldn't be.